nix/pwndbg.nix

```diff
@@ -71,13 +51,18 @@ let
       mkdir -p $out/share/pwndbg
 
       cp -r gdbinit.py pwndbg $out/share/pwndbg
+      # Build self-contained init script for lazy loading from vanilla gdb
+      # I purposely use insert() so I can re-import during development without having to restart gdb
+      sed "2 i import sys, os\n\
+      sys.path.insert(0, '${pyEnv}/${pyEnv.sitePackages}')\n\
+      sys.path.insert(0, '$out/share/pwndbg/')\n\
+      os.environ['PATH'] += ':${binPath}'\n\
+      os.environ['LC_CTYPE'] = 'C.UTF-8'\n" -i $out/share/pwndbg/gdbinit.py
```

Comment:
@disconnect3d i think this need to be upstreamed into `gdbinit.py` not only for nix'ed version

Reply:
I think we already fixed it in https://github.com/pwndbg/pwndbg/pull/2139
Se `LC_CTYPE` can be removed.

Reply:
fixed